modules/cloud-auth/cloud-auth-grammar.ym

```diff
@@ -55,6 +56,12 @@ CloudAuthenticator *last_authenticator;
 %token KW_USER_MANAGED_SERVICE_ACCOUNT
 %token KW_NAME
 %token KW_METADATA_URL
+%token KW_AZURE
+%token KW_AZURE_MONITOR
```

Comment:
Can we rename this to `KW_MONITOR` so it does not cause confusion that the keyword itself is `"monitor"`?